app/views/admin/pubsubhubbub/index.html.haml

```diff
@@ -11,7 +11,7 @@
       %th= t('admin.pubsubhubbub.last_delivery')
   %tbody
     - @subscriptions.each do |subscription|
-      - expired = Time.now.utc < subscription.expires_at
+      - expired = Time.now.utc > subscription.expires_at
```

Comment:
Can you put this into a method on subscription, instead of setting a variable in a view?